control/xferfcn.py

```diff
@@ -10,6 +10,7 @@
 # Python 3 compatibility (needs to go here)
 from __future__ import print_function
 from __future__ import division
+from __future__ import absolute_import
```

Comment:
Had to look this up.  It appears to be necessary for Python 2.5 - do we support that?  I didn't quite understand what the related PEP was getting at, but it's not obvious to me that your changes especially need the features it enables anyway.